src/Form/Mapper/FormMapper.php

```diff
@@ -21,56 +21,30 @@
 interface FormMapper
 {
     /**
-     * @param string $name
-     * @param mixed  $type
-     *
-     * @return FormBuilderInterface
+     * @param mixed $type
      */
-    public function create($name, $type = null, array $options = []);
+    public function create(string $name, $type = null, array $options = []): FormBuilderInterface;
```

Comment:
```suggestion
    public function create(string $name, ?string $type = null, array $options = []): FormBuilderInterface;
```
It make no sense to pass an object here. It will call symfony `FormBuilder::create`
https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Form/FormBuilderInterface.php#L34-L43